Core/scripts/dirac-install.py

```diff
@@ -741,72 +1053,89 @@ def getModSource(self, release, modName, project=None):
       return S_OK((False, modTpl[0]))
     return S_OK((modTpl[0], modTpl[1]))
 
-  def getExtenalsVersion(self, release=False):
-    if 'DIRAC' not in self.__prjRelCFG:
+  def getExtenalsVersion(self, release=None):
+    """
+    It returns the version of DIRAC Externals. If it is not provided,
+    uses the default cfg
+
+    :param str release: the release version
+    """
+
+    if 'DIRAC' not in self.prjRelCFG:
       return False
     if not release:
-      release = list(self.__prjRelCFG['DIRAC'])
+      release = list(self.prjRelCFG['DIRAC'])
       release = max(release)
     try:
-      return self.__prjRelCFG['DIRAC'][release].get('Releases/%s/Externals' % release)
+      return self.prjRelCFG['DIRAC'][release].get('Releases/%s/Externals' % release)
     except KeyError:
       return False
 
-  def getDiracOSVersion(self, diracOSVersion=''):
+  def getDiracOSVersion(self, diracOSVersion=None):
+    """
+    It returns the DIRAC os version
+    :param str diracOSVersion: the OS version
+    """
+
     if diracOSVersion:
       return diracOSVersion
     try:
-      return self.__prjRelCFG[self.__projectName][cliParams.release].get(
+      return self.prjRelCFG[self.projectName][cliParams.release].get(
           "Releases/%s/DiracOS" % cliParams.release, diracOSVersion)
     except KeyError:
       pass
     return diracOSVersion
 
-  def getLCGVersion(self, lcgVersion=""):
+  def getLCGVersion(self, lcgVersion=None):
+    """
+    It returns the LCG version
+    :param str lcgVersion: LCG version
+    """
     if lcgVersion:
       return lcgVersion
     try:
-      return self.__prjRelCFG[self.__projectName][cliParams.release].get(
+      return self.prjRelCFG[self.projectName][cliParams.release].get(
           "Releases/%s/LcgVer" % cliParams.release, lcgVersion)
     except KeyError:
       pass
     return lcgVersion
 
-  def getModulesToInstall(self, release, extensions=False):
+  def getModulesToInstall(self, release, extensions=None):
+    """
+    It returns the modules to be installed.
```

Comment:
Here, and in the previous ones, where there a phrase like " It returns the ..something..", just use `:returns:`